dev-proxy/PluginLoader.cs

```diff
@@ -45,8 +45,11 @@ public PluginLoaderResult LoadPlugins(IPluginEvents pluginEvents, IProxyContext
                 Assembly assembly = pluginLoadContext.LoadFromAssemblyName(new AssemblyName(Path.GetFileNameWithoutExtension(pluginLocation)));
                 IEnumerable<UrlToWatch>? pluginUrlsList = h.UrlsToWatch?.Select(ConvertToRegex);
                 ISet<UrlToWatch>? pluginUrls = null;
-                if (pluginUrlsList is not null)
-                {
+                if (pluginUrlsList is null || !pluginUrlsList.Any()){
```

Comment:
Perfect! Let's move { to the next line for consistent formatting and we'll get it merged! We appreciate your help!

Reply:
> Perfect! Let's move { to the next line for consistent formatting and we'll get it merged! We appreciate your help!

```
if (pluginUrlsList is not null && !pluginUrlsList.Any())
        {
            // Set pluginUrlsList to globallyWatchedUrls.ToList() if it's null or empty
            pluginUrlsList = globallyWatchedUrls.ToList();
        }
        else if (pluginUrlsList?.Any() == true)
        {
            // Directly check if pluginUrlsList is not null and contains elements
            pluginUrls = pluginUrlsList.ToHashSet();
            globallyWatchedUrls.AddRange(pluginUrlsList);
        }
```